_includes/v3.11/release-notes/v3.11.0-release-notes.md

```diff
@@ -0,0 +1,36 @@
+18 Dec 2019
+
+#### Support dual-stack pod IPs
+
+Native support for both IPv4 and IPv6 is known as "dual stack" and Kubernetes has alpha-level support for this in versions 1.16 and 1.17.
+With this v3.11 release, Calico now supports dual stack. This allows you to use both IPv4 and IPv6 for your cluster pods. For more information on enabling
```

Comment:
"now supports the new Kubernetes APIs for dual-stack"

The main difference is that Calico has had dual-stack support for a while, the thing we added was an integration with k8s

Reply:
Thanks, fixed e4341771209c6d48753d1507c2bda61e444df738